map.py

```diff
@@ -9,20 +9,8 @@ def __init__(self, num_of_points, map_size, seed):
         self.seed = seed
         self.points = set()
 
-    def generate_map(self):
-        """
-        A function that generates random points with their coordinates and limits
-        with Map objet's parametars
+    def _update_limits(self):
```

Comment:
Mislio sam da samo ovo `if` grananje bude pomocna funkcija pa da imas kod:
```
for point1, point2 in points:
    _update_limits(point1, point2)
```

Ok je i ovako, samo bih preimenovao funkciju u `_update_point_limits`, jer kad zoves `map._update_limits` to govori da apdejtujes limite za mapu, a ne za limite njenih tacaka.